docs/cluster_spec.md

```diff
@@ -499,6 +499,15 @@ spec:
     protectKernelDefaults: true
 ```
 
+### Housekeeping Interval
+Interval between container housekeepings which defaults to `10s`. This can be high for some use cases and can be modified by setting kubelet flag `--housekeeping-interval`.
```

Comment:
```suggestion
{{ kops_feature_table(kops_added_default='1.19', k8s_min='1.2') }}

The interval between container housekeepings defaults to `10s`. This can be too small or too high for some use cases and can be modified with the following addition to the kubelet spec.
```